### ISSUE TITLE
Cодержимое

### ISSUE THREAD

**Issue by Mihrutkin** [remittor/zapret-openwrt#21]
Удалил содержимое NFQWS_OPT
Поделитесь исходным кодом (не дает удалить пакет, чтобы переустановить)

**Comment by Mihrutkin** [comment]
Имею ввиду разделом NFQWS_OPT

**Comment by remittor** [comment]
Нажмите reset и всё.

**Comment by Mihrutkin** [comment]
Кнопки стали неактивны 
![Снимок](https://github.com/user-attachments/assets/8a8e8c7d-da4a-40e7-a3ea-1f136a1388e6)

**Comment by remittor** [comment]
Кнопки неактивны, т.к. возникла ошибка. Вам об этом сообщают выше.
Попробуйте удалить ipk и поставить заново.

**Comment by Mihrutkin** [comment]
Да я же сверху написал, что не могв удалить
![Uploading Screenshot_2024-11-12-21-11-45-314.jpeg…]()

**Comment by Mihrutkin** [comment]
![Screenshot_2024-11-12-21-11-45-314](https://github.com/user-attachments/assets/e39949b5-576e-40cf-9400-f52f61b46514)

**Comment by Mihrutkin** [comment]
Помогло, спасибо

**Comment by remittor** [comment]
Да. Странная ошибка. Видимо после мержинга конфигов ломается синтаксис основного файла конфига /opt/zapret/config

Жаль что этот корявый config вы мне не показали. Поэтому первоисточник проблемы ещё не найден.

Но в процесс деинсталяции я добавил доп. проверку:
https://github.com/remittor/zapret-openwrt/commit/8d9a30f2eeb18bb22d6e7927e5c6cfc537b9b085

**Comment by Mihrutkin** [comment]
Отбой, ресет помог

**Comment by remittor** [comment]
Думаю вот этот фикс исправит сию траблу: https://github.com/remittor/zapret-openwrt/commit/b0953bbaaf5b05ec8908de50e99a567d83bdc92a

**Comment by drmirro** [comment]
На версии V68-20241110 , после очередного редактирования конфига столкнулся с подобными сообщениями и неактивными кнопками.
rm -f /usr/lib/opkg/info/zapret.* и далее opkg update не помогло.
Снес все полностью и установил V68-20241115. Поставил свой конфиг  с которым у меня ЮТ , кое-как, работает еще с V67.
ЮТ не заработал . Пробовал опять поставить конфиг "из коробки" через  Reset settings- выскочил запрос на подтверждение сброса с неактивными кнопками и ничего не поменялось. Опять все снес и вновь поставил V68-20241110. Удалил дефолтный конфиг ,поставил свой -заработало ,

**Comment by Miraihi** [comment]
Та же проблема. Вылечилась удалением лишней кавычки в `/opt/zapret/config` в строке `NFQWS_OPT=`
